### PR TITLE
Sprint4 player setup

### PR DESCRIPTION
**Pull request by Bhavani-Chanda**
Added destabilizers and test files for sprint_four_player creation.

### CODE REVIEW

File: src/sprint_four_player/Headquarters.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
+package sprint_four_player;
+
+import battlecode.common.*;
+import sprint_three_player.Communication;
```
Comment (jconnorbw) [inline src/sprint_four_player/Headquarters.java]: This should be sprint_four_player. However, this import statement is not needed at all. I would say remove it. But, if it is kept, change it to sprint_four_player.

**Review by jconnorbw — CHANGES_REQUESTED** [review]
Looks great! Just need to change or remove the statement on line 4 in the Headquarters.java file.

**Review by swethasrihari — APPROVED** [review]
I agree with Connor. Please change or remove the import statement for Communication in the Headquarters class and we're good to go.

File: src/sprint_four_player/Headquarters.java (hunk 1 above)
Comment (Bhavani-Chanda) [reply src/sprint_four_player/Headquarters.java]: Done! removed the requested change.

**Pull request merged**